.README.html

```diff
@@ -214,6 +214,9 @@ <h1 id="role-variables">Role Variables</h1>
 the upper limit of messages from a service which are handled within the
 time defined by <code>journald_rate_limit_interval_sec</code>. See
 <code>man 5 journald.conf</code> for more information.</p></li>
+<li><p><code>journald_max_retention</code> - integer variable, in
+minutes, sets how long journal entries can be retained before they are deleted.
+No implicit value is configured by the role.</p></li>
```

Comment:
The README.html is automatically generated from README.md - please make any documentation changes to README.md